apps/web/vibes/soul/data/locales.ts

```diff
@@ -0,0 +1,6 @@
+export const locales = [
+  { id: 'US', label: 'United States' },
+  { id: 'FR', label: 'France' },
+  { id: 'DE', label: 'Denmark' },
+  { id: 'IT', label: 'Italy' },
```

Comment:
These are invalid locale identifiers. Locale identifiers require a language code while the country code is optional. Maybe you meant to use `en`, `fr`, `de`, and `it`?